docs/static/monitoring-apis.asciidoc

```diff
@@ -333,22 +370,46 @@ Example response:
 [source,js]
 --------------------------------------------------
 {
-  "process": {
-    "open_file_descriptors": 164,
-    "peak_open_file_descriptors": 166,
-    "max_file_descriptors": 10240,
-    "mem": {
-      "total_virtual_in_bytes": 5399474176
+  "process" : {
+    "open_file_descriptors" : 184,
+    "peak_open_file_descriptors" : 185,
+    "max_file_descriptors" : 10240,
+    "mem" : {
+      "total_virtual_in_bytes" : 5486125056
     },
-    "cpu": {
-      "total_in_millis": 72810537000,
-      "percent": 0,
-      "load_average": {
-        "1m": 2.41943359375
+    "cpu" : {
+      "total_in_millis" : 657136,
+      "percent" : 2,
+      "load_average" : {
+        "1m" : 2.38134765625
       }
     }
   }
-}
+--------------------------------------------------
+
+[[event-stats]]
+==== Event Stats
+
+The following request returns a JSON document containing cumulative event counts
```

Comment:
maybe `event related statistics` instead of `event counts`, as duration_in_millis and queue_push_duration_in_millis is also displayed?